File: Homeworks/C# Part 1/04.ConsoleInAndOut/06.QuadraticEquation/QuadraticEquation.cs
using System;

class QuadraticEquation
{
    static void Main()
    {
        double numberA = double.Parse(Console.ReadLine());
        double numberB = double.Parse(Console.ReadLine());
        double numberC = double.Parse(Console.ReadLine());
        double determinant = (numberB * numberB) - (4 * numberA * numberC);
        double rootOne = (-numberB + Math.Sqrt(determinant)) / (double)(2 * numberA);
        double rootTwo = (-numberB - Math.Sqrt(determinant)) / (double)(2 * numberA);
        if (determinant > 0)
        {
            Console.WriteLine("{0:F2}", Math.Min(rootOne, rootTwo));
            Console.WriteLine("{0:F2}", Math.Max(rootOne, rootTwo));
        }
        else if (determinant == 0)
        {
            Console.WriteLine("{0:F2}", rootOne);
        }
        else
        {
            Console.WriteLine("no real roots");
        }
    }
}

File: Homeworks/JS Fundamentals/05. Conditional Statements/06. Quadratic equation/quadratic-equation.js
function solve(args) {
	var a = +args[0];
	var b = +args[1];
	var c = +args[2];
	var determinant = (b * b) - (4 * a * c);
	var rootOne = (-b + Math.sqrt(determinant)) / (2 * a);
    var rootTwo = (-b - Math.sqrt(determinant)) / (2 * a);
    if (determinant > 0) {
        console.log('x1=' + Math.min(rootOne, rootTwo).toFixed(2) + '; x2=' + Math.max(rootOne, rootTwo).toFixed(2));        
    }
    else if (determinant == 0) {
        console.log('x1=x2=' + rootOne.toFixed(2));
    } else {
        console.log("no real roots");
    }
}